### DISCUSSION TITLE
Logo for R.nvim?

### DISCUSSION THREAD

**Discussion by jalvesaq** [R-nvim/R.nvim#89] [General]
We got a generic checkboard-like pattern as the `R-nvim` organization "logo". Perhaps someone could draw a real original logo. I mixed R and Neovim logos, but my artistic skills are below average and I'm sure that anyone can do a better job:

![R nvim](https://github.com/R-nvim/R.nvim/assets/891655/59244fc4-8108-4fb8-a95a-9e25cae8e383)

In the above logo, it seems that R and Neovim are unwillingly shackled and not productively connected.

**Comment by PMassicotte** [comment]
ahah so nice! I was just about to ask you :) REALLY like it!

**Comment by PMassicotte** [comment]
Want me to ask my artist friend to work on your suggestion?

**Comment by jalvesaq** [comment]
Yes, please, if your friend could work for free...

**Comment by PMassicotte** [comment]
I have not yet received news from the person I contacted. If you want, we can go with the logo you made until she gets back to me.

**Comment by jalvesaq** [comment]
I replaced Neovim "flat" logo with the "mark" one:

![R nvim2](https://github.com/R-nvim/R.nvim/assets/891655/a5f19a2b-1ed3-4dfc-b060-7fe51824beed)

I think it's better because the grey ring was free to pass through the gap between the "N" bars.

**Reply by PMassicotte** [reply]
I prefer this one.

**Comment by jalvesaq** [comment]
I think we can use this second proposal temporarily.

**Reply by PMassicotte** [reply]
Excellent job!

**Comment by jalvesaq** [comment]
Thanks! Using the provisory logo now.
